ush/setup.py

```diff
@@ -1496,10 +1414,11 @@ def dict_find(user_dict, substring):
     # (e.g. metatasks with no tasks, tasks with no associated commands)
     clean_rocoto_dict(expt_config["rocoto"]["tasks"])
 
-    rocoto_yaml_fp = workflow_config["ROCOTO_YAML_FP"]
-    with open(rocoto_yaml_fp, 'w') as f:
-        yaml.Dumper.ignore_aliases = lambda *args : True
-        yaml.dump(expt_config.get("rocoto"), f, sort_keys=False)
+    rocoto_yaml_fp = Path(workflow_config["ROCOTO_YAML_FP"])
+    rocoto_yaml_dict = expt_config["rocoto"]
+    extend_yaml(rocoto_yaml_dict)
+    rocoto_yaml = get_yaml_config(rocoto_yaml_dict)
+    rocoto_yaml.dump(rocoto_yaml_fp)
```

Comment:
Jump through a few hoops to get Rocoto dict into a UWConfig object. Precursor to actually transitioning to uw rocoto tool.